core/src/main/kotlin/fr/sncf/osrd/stdcm/preprocessing/implementation/BlockAvailability.kt

```diff
@@ -340,6 +345,7 @@ fun makeBlockAvailability(
     steps: List<STDCMStep> = listOf(),
     gridMarginBeforeTrain: Double = 0.0,
     gridMarginAfterTrain: Double = 0.0,
+    timeStep: Double = 2.0,
```

Comment:
I'd prefer if it's renamed here, it doesn't function as a time step. Maybe something like `plannedTimingTolerance`?